tests/datastore/test_database.py

```diff
@@ -0,0 +1,221 @@
+import pytest
+import os
+import uuid
+from itsim.datastore.database import DatabaseSQLite
+from itsim.schemas.itsim_items import create_json_item
+from itsim.time import now_iso8601
+from contextlib import contextmanager
+
+
+@contextmanager
+def db_file():
+    sqlite_file_name = '.sqlite'
+    try:
+        if os.path.isfile(sqlite_file_name):
+            os.remove(sqlite_file_name)
```

Comment:
If you keep this, then at the beginning of any one of your tests, it is reasonable to expect the file database not to exist. In particular, at the end of every test, you have the certainty of removing it, thanks to this context manager.
```suggestion
            pytest.fail("Database file should not exist.")
```